Lib/poplib.py

```diff
@@ -451,7 +451,6 @@ def stls(self, context=None):
     __all__.append("POP3_SSL")
 
 if __name__ == "__main__":
-    import sys
```

Comment:
I'm not sure about this one; an alternative could be to adjust the top-level `import`.

```diff
diff --git a/Lib/poplib.py b/Lib/poplib.py
index 9a5ef03c98..17a929b788 100644
--- a/Lib/poplib.py
+++ b/Lib/poplib.py
@@ -16,7 +16,7 @@
 import errno
 import re
 import socket
-import sys
+from sys import audit
 
 try:
     import ssl
@@ -100,7 +100,7 @@ def __init__(self, host, port=POP3_PORT,
         self.host = host
         self.port = port
         self._tls_established = False
-        sys.audit("poplib.connect", self, host, port)
+        audit("poplib.connect", self, host, port)
         self.sock = self._create_socket(timeout)
         self.file = self.sock.makefile('rb')
         self._debugging = 0
@@ -113,7 +113,7 @@ def _create_socket(self, timeout):
 
     def _putline(self, line):
         if self._debugging > 1: print('*put*', repr(line))
-        sys.audit("poplib.putline", self, line)
+        audit("poplib.putline", self, line)
         self.sock.sendall(line + CRLF)
 
```

Reply:
What do you think?

Reply:
I prefer using `sys.audit`; it's not particularly verbose, and using the fully-qualified name means you don't have to scan the file to find out where this "audit" function came from when you're reading through the code :)

Reply:
All right, let's remove the `__main__` import, then :)